### PR TITLE
(#108) Use EL7 packages for EL8 and Fedora.

### PR DESCRIPTION
**Pull request by silug**

### CODE REVIEW

File: manifests/repo.pp [hunk 1]
```diff
@@ -8,10 +8,17 @@
   assert_private()
 
   if $facts["os"]["family"] == "RedHat" {
+    if versioncmp($facts['os']['release']['major'], '5') < 0 {
+      fail("Choria Repositories are only supported for RHEL/CentOS 5 or newer releases")
```
Comment (ripienaar) [inline manifests/repo.pp]: we can drop el5 support, I cannot continue to build choria binaries for it as Go doesnt support it. So el5 choria is dead.

File: manifests/repo.pp [hunk 2]
```diff
@@ -8,10 +8,17 @@
...
+    } elsif versioncmp($facts['os']['release']['major'], '7') > 0 {
+      $release = '7'
```
Comment (ripienaar) [inline manifests/repo.pp]: ok, though package cloud now supports el8 so we can build el8 packages - though without centos 8 i cant test really.  Don't know if you have interest in adding el8 packages, I can guide you on that.

File: manifests/repo.pp (hunk 1 above)
Comment (silug) [reply manifests/repo.pp]: That's completely reasonable.  I'll push an update to drop EL 5 support.

File: manifests/repo.pp (hunk 2 above)
Comment (silug) [reply manifests/repo.pp]: Without CentOS 8, I don't know that I'd bother.  For now, the EL 7 package seems to work fine on RHEL 8 and recent Fedora versions.

**Comment by ripienaar** [comment]
thank you

**Pull request merged**